eq-author/src/components/ContentPickerv2/Menu.js

```diff
@@ -0,0 +1,253 @@
+  border-bottom: 1px solid #d8d8d8;
+  margin: -1px 0;
+  width: 100%;
+  overflow: hidden;
+  transition: all 100ms ease-in-out;
+
+  &[disabled] {
+    opacity: 0.5;
+    pointer-events: none;
+  }
+
+  &:focus {
+    outline: 2px solid ${colors.tertiary};
+    z-index: 4;
+  }
+`;
+
+export const ParentMenuItem = styled(MenuItem)`
+  padding: 0 1em;
+
+  &::after {
+    content: "";
+    position: absolute;
+    background: url(${iconChevron});
+    width: 16px;
+    height: 16px;
+    right: 0.5em;
+    opacity: 0.5;
+  }
+
+  &[aria-selected="true"] {
+    background: #f3f3f3;
+    &::after {
+      opacity: 1;
+    }
+    &:hover {
+      background: #f3f3f3;
+    }
+  }
+`;
+
+export const SubMenuItem = styled(MenuItem)`
+  --color: ${colors.text};
+  --colorSecondary: ${colors.textLight};
+  --colorTertiary: ${colors.text};
+
+  &[aria-selected="true"] {
+    --color: ${colors.white};
+    --colorSecondary: ${colors.white};
+
+    background: ${colors.primary};
+    border-color: #377090;
+    z-index: 2;
+
+    &:hover {
+      background: #397596;
+    }
+  }
+`;
+
+export const MenuItemTitles = styled.div`
+  flex: 1 1 auto;
+  overflow: hidden;
+  margin-right: 0.5em;
+`;
+
+export const MenuItemTitle = styled.div`
+  font-size: 1em;
+  margin-bottom: 0.1em;
+  color: var(--color);
+`;
+
+export const MenuItemSubtitle = styled.div`
+  font-size: 0.9em;
+  color: var(--colorSecondary);
+`;
+
+export const MenuItemType = styled.span`
+  font-size: 10px;
+  background: #e4e8eb;
+  padding: 0.3em 0.7em;
+  border-radius: 1em;
+  text-transform: uppercase;
+  letter-spacing: 0.05em;
+  color: var(--colorTertiary);
+  flex: 0 1 auto;
+  justify-self: flex-end;
+`;
+
+const SectionTitle = styled.div`
+  background: #f4f5f6;
+  font-size: 11px;
+  padding: 0.3rem 1rem;
+  text-transform: uppercase;
+  letter-spacing: 0.05em;
+  font-weight: bold;
+  color: #666;
+  line-height: 1.1;
+
+  &:not(:first-of-type) {
+    border-top: 1px solid #e4e8eb;
+  }
+
+  &:not(:last-of-type) {
+    border-bottom: 1px solid #e4e8eb;
+  }
+`;
+
+const Menu = ({ data, onSelected, isSelected }) => {
+  const onEnterUp = (event, section) => {
+    if (event.keyCode === 13) {
+      //13 is the enter keycode
+      onSelected(section);
+    }
+  };
+
+  return (
+    <MenuItemList>
+      {data &&
+        data.map(section => {
+          const sectionSelected = isSelected(section);
+
+          return (
+            <ParentMenuItem
+              key={section.id}
+              onClick={() => {
+                onSelected(section);
+              }}
+              aria-selected={sectionSelected}
+              tabIndex={sectionSelected ? "" : "0"}
+              onKeyUp={event => onEnterUp(event, section)}
+            >
+              {section.displayName}
+            </ParentMenuItem>
+          );
+        })}
+    </MenuItemList>
+  );
+};
+
+Menu.propTypes = {
+  data: PropTypes.arrayOf(
+    PropTypes.shape({
+      id: PropTypes.string.isRequired,
```

Comment:
Sorry missed these earlier, can we use the custom-prop type here as well, helps to understand the component at a glance.